washboard-ui/packages/washboard-ui/src/app/contexts/settings-context.tsx

```diff
@@ -10,16 +11,25 @@ enum DarkModeOption {
 export type SettingsContextValue = {
   darkMode: DarkModeOption;
   setDarkMode: (darkMode: DarkModeOption) => void;
+  wadmManagedAsset: WadmManagedAssetOption;
+  setWadmManagedAsset: (wadmManagedAsset: WadmManagedAssetOption) => void;
 };
 
 export const SettingsContext = createContext<SettingsContextValue>({
   darkMode: DarkModeOption.System,
   setDarkMode: () => null,
+  wadmManagedAsset: WadmManagedAssetOption.None,
+  setWadmManagedAsset: () => null,
 });
 
 export function SettingsProvider({children}: PropsWithChildren): ReactElement {
   const [darkMode, setDarkMode] = useLocalStorage('theme', DarkModeOption.System);
 
+  const [wadmManagedAsset, setWadmManagedAsset] = useLocalStorage(
+    'wadmManagedAsset',
+    WadmManagedAssetOption.None,
+  );
```

Comment:
```suggestion
export const SettingsContext = createContext<SettingsContextValue>({
  darkMode: DarkModeOption.System,
  setDarkMode: () => null,
  wadmManagedAsset: WadmManagedAssetOption.Logo,
  setWadmManagedAsset: () => null,
});

export function SettingsProvider({children}: PropsWithChildren): ReactElement {
  const [darkMode, setDarkMode] = useLocalStorage('theme', DarkModeOption.System);

  const [wadmManagedAsset, setWadmManagedAsset] = useLocalStorage(
    'wadmManagedAsset',
    WadmManagedAssetOption.Logo,
  );
```

Let's go with the logo as default! 💥